### PR TITLE
[14.0][IMP] ddmrp:  Allow inheritance in the report BoM Lead Time

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

File: ddmrp/report/mrp_report_bom_structure.xml [hunk 1]
```diff
@@ -31,7 +31,8 @@
             >
                 <span>
                     <t
-                        t-esc="data['bom'].product_tmpl_id.produce_delay"
+                        t-if="'lead_time' in data"
```
Comment (LoisRForgeFlow) [inline ddmrp/report/mrp_report_bom_structure.xml]: maybe you can avoid the t-if?

Comment (BernatPForgeFlow) [reply ddmrp/report/mrp_report_bom_structure.xml]: Done!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**